src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt

```diff
@@ -43,27 +42,32 @@ class HentaiSlayer : ParsedHttpSource(), ConfigurableSource {
     }
 
     // ============================== Popular ===============================
-    override fun popularMangaRequest(page: Int) = GET("$baseUrl/manga?page=$page", headers)
-
-    override fun popularMangaSelector() = "div > div:has(div#card-real)"
-
-    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        element.selectFirst("div#card-real a")?.run {
-            setUrlWithoutDomain(absUrl("href"))
-            selectFirst("figure")?.run {
-                selectFirst("img.object-cover")?.run {
-                    thumbnail_url = imgAttr()
-                    title = attr("alt")
+    override fun popularMangaRequest(page: Int) =
+        GET("$baseUrl/manga?page=$page", headers)
+
+    override fun popularMangaSelector() =
+        "div > div:has(div#card-real)"
+
+    override fun popularMangaFromElement(element: Element) =
+        SManga.create().apply {
+            element.selectFirst("div#card-real a")?.run {
+                setUrlWithoutDomain(absUrl("href"))
+                selectFirst("figure")?.run {
+                    selectFirst("img.object-cover")?.run {
+                        thumbnail_url = imgAttr()
+                        title = attr("alt")
+                    }
+                    genre = select("span p.drop-shadow-sm").text()
```

Comment:
```suggestion
            element.selectFirst("div#card-real a")!!.run {
                setUrlWithoutDomain(absUrl("href"))
                selectFirst("figure")!!.run {
                    selectFirst("img.object-cover")!!.run {
                        thumbnail_url = imgAttr()
                        title = attr("alt")
                    }
                    genre = select("span p.drop-shadow-sm").text()
```
title and url is mandatory and should be populated

Reply:
And replace `run()` with `with()`.